app/views/layouts/overdue.html.erb

```diff
@@ -5,23 +5,13 @@
   <p>
     Patients that are overdue for a follow-up visit<% if current_admin.accessible_facilities(:manage_overdue_list) %>. If you select a facility, you can download a patient list.<% end %>
   </p>
-  <% if current_admin.accessible_facilities(:manage_overdue_list) %>
-    <%= render('shared/paginate_and_filter_by_facility',
-               form_url: appointments_path,
-               current_admin: current_admin,
-               facility_id: @facility_id,
-               per_page: @per_page,
-               form_id: "search-filters",
-               scope_namespace: current_admin.accessible_facilities(:manage_overdue_list)) %>
-  <% else %>
```

Comment:
What was this `else` branch for?

Reply:
I temporarily force-executed the `else` branch, it errored out. The `scope_namespace` doesn't make sense for how it's used in the partial.

This might have been an access-related change, but this page will correctly show empty results if you don't have access to any facilities, or kick you off completely if you don't have PII access in the first place.

Reply:
Hmm, odd